pages/set-root-password-page.ts

```diff
@@ -0,0 +1,24 @@
+// Set root password before installation
+
+export class SetRootPasswordPage {
+  readonly page: Page;
+  readonly password: Locator;
+  readonly confirmPassword: Locator;
+  readonly confirmButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.password = page.getByRole('textbox', { name: 'User password', exact: true });
+    this.confirmPassword = page.getByLabel('Password confirmation');
+    this.confirmButton = page.getByRole('button', { name: 'Confirm' });
+  }
+  
+  async fillPassword(password: string) {
```

Comment:
if we test errors in this screen, it is not clear that filling password (singular) we can achieve both fields, you can group it, but the name should reflect that, ie: `fillPasswordAndConfirm`